packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/bootstrapping.integtest.ts

```diff
@@ -82,6 +82,24 @@ integTest('can and deploy if omitting execution policies', withoutBootstrap(asyn
   });
 }));
 
+// Custom Bootstrap test with Session Tags on the DeployRole
+integTest('can deploy with session tags on the deploy role', withoutBootstrap(async (fixture) => {
```

Comment:
As we discussed, at a test for a custom synthesizer that doesn't define a cloudformation execution role and defines session tags for the deploy role.